tests/dao_tests/test_participant_dao.py

```diff
@@ -506,6 +506,31 @@ def test_error_log_for_incorrect_status_on_participant(self, mock_logging):
             f'Un-consented participant {participant.participantId} was withdrawn with NO_USE'
         )
 
+    def test_return_withdrawn_ids_only(self):
+        participants_status = {
+            1: WithdrawalStatus.NO_USE,
+            2: WithdrawalStatus.NOT_WITHDRAWN,
+            3: WithdrawalStatus.NOT_WITHDRAWN,
+            4: WithdrawalStatus.EARLY_OUT
+        }
+        expected_result = [
+            str(pid)
+            for pid, status in participants_status.items()
+            if status in [WithdrawalStatus.NO_USE, WithdrawalStatus.EARLY_OUT]
+        ]
+
+        # Insert into database
+        for pid, status in participants_status.items():
+            bid = pid
+            self.dao.insert(
+                Participant(
+                    participantId=pid, biobankId=bid, withdrawalStatus=status
+                )
+            )
+        result = self.dao.get_withdrawn_participant_ids(list(participants_status.keys()))
+
+        self.assertEqual(result, expected_result)
```

Comment:
TBH the more tests the better but moving forward I think we can not add tests for utility methods in the daos unless its super specific. Can we also edit this to lean into the provided generators?